examples-migrator/examples-migrator.js

```diff
@@ -54,7 +66,16 @@ examplePaths.forEach(newPath => {
   fs.copySync(path.resolve(OLD_EXAMPLES_PATH, pathMap[newPath]), path.resolve(NEW_EXAMPLES_PATH, newPath));
 });
 
-// Copy our custom gitignore and replace the other one
+// Move the boilerplate to the tools folder
+let boilerplate = globby.sync(FILES_TO_IGNORE, { cwd: OLD_EXAMPLES_PATH, dot: true });
+boilerplate.forEach(newPath => {
+  const oldPath = newPath;
+  if (newPath.startsWith('_')) newPath = newPath.substr(1);
```

Comment:
I would have been tempted to swap the param with the variable:

```
boilerplate.forEach(oldPath => {
  const newPath = (oldPath.startsWith('_') ? oldPath.substr(1) : oldPath;
...
```

Since it is the old path that we are iterating over.

But it is not a big issue.